backend/src/handlers/alert.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from fastapi import APIRouter, Form, Request
+from fastapi.responses import JSONResponse
+from src.model.alert import Alert
+
+router = APIRouter(prefix="/alerts", tags=["alerts"])
+
+
+@router.post("/")
+def post_alert(
+    req: Request, text: str = Form(), start_time: int = Form(), end_time: int = Form()
+) -> JSONResponse:
+    with req.app.state.db.session() as session:
+        dt_start_time = datetime.datetime.fromtimestamp(start_time)
```

Comment:
nit: why prefix with dt?

Reply:
Didn't want to conflict with the variable passed in, but I realize it is a sucky name